modules/psap-driver-toolkit-using.adoc

```diff
@@ -68,9 +68,6 @@ spec:
 
       WORKDIR /build/
 
-      RUN yum -y install git make sudo gcc \
-      && yum clean all \
-      && rm -rf /var/cache/dnf
 
```

Comment:
Delete the double space line.

Reply:
I may have been misunderstood... I just meant to delete the double space line, but one space line should still remain. It's not critical to anything, but would look nicer. Thanks.